### PR TITLE
Update __init__.py

### PR DESCRIPTION
**Pull request by SmashedFrenzy16**

### CODE REVIEW

File: src_py/__init__.py [hunk 1]
```diff
@@ -335,11 +335,11 @@ def __color_reduce(c):
 # Thanks for supporting pygame. Without support now, there won't be pygame later.
 if "PYGAME_HIDE_SUPPORT_PROMPT" not in os.environ:
     print(
-        "pygame {} (SDL {}.{}.{}, Python {}.{}.{})".format(  # pylint: disable=consider-using-f-string
+        "Pygame v{} (SDL v{}.{}.{}, Python v{}.{}.{})".format(  # pylint: disable=consider-using-f-string
```
Comment (novialriptide) [inline src_py/__init__.py:338]: Lowercase p is usually preferred when referring to "pygame" as per PEP8
https://peps.python.org/pep-0008/#package-and-module-names
image

**Review by CodeMaster7000 — CHANGES_REQUESTED** [review]
Leave a thanks message to those who contribute. You could say something along the lines of "We appreciate you taking the time to contribute."

**Comment by SmashedFrenzy16** [comment]
@novialriptide, I have lower cased pygame

**Comment by SmashedFrenzy16** [comment]
@CodeMaster7000, wouldn't that make the message too long? After all, it is displayed when you run a pygame project. But I will try and include something relevant to what you are saying.

**Comment by novialriptide** [comment]
It's also notable to leave meaningful commit messages instead of just "Update filename"

**Comment by Starbuck5** [comment]
Closed via: https://github.com/pygame/pygame/pull/3640#issuecomment-1368308737

**Pull request closed without merging**